new-package/elm-review-package-tests/check-previews-compile.js

```diff
@@ -82,12 +82,13 @@ function checkDepsAreCompatible(exampleConfiguration, previewDependencies) {
       previewDependencies[depName]
     );
     delete previewDependencies[depName];
-  });
+  }
 
   const remainingKeys = Object.keys(previewDependencies);
-  if (remainingKeys.length !== 0) {
+  if (remainingKeys.length > 0) {
     console.error(
-      `There are extraneous dependencies in the ${exampleConfiguration}/ configuration: ${remainingKeys}`
+      // prettier-ignore
```

Comment:
I have to admit I'm not fond in having disable checks (for prettier or for TS) in files that package authors will then own.
Maybe in this case, we can create a new variable `const keys = remainingKeys.join(', ');` above `console.error(...)`?
That should get rid of the need for the ignore.

Reply:
You marked this as resolved, but the comment is still here. Did you apply it on a different branch maybe?

Reply:
Let's resolve this later, let's not block all the other PRs because of this.

Reply:
Huh. Well, once it's all merged I can check again.